Short-Answer/Algorithms_Questions.md

```diff
@@ -7,26 +7,25 @@ pseudocode with respect to the input size n of each of the following:
 
 ```python
 a)  a = 0
-    while (a < n * n * n):
-      a = a + n * n
+    while (a < n * n * n): # n^3
+      a = a + n * n #n^2
```

Comment:
Great work commenting the run-times on each line then calculating the full run-time based on how the combine.